internal/promotion/runner/builtin/git_cloner_test.go

```diff
@@ -115,6 +115,57 @@ func Test_gitCloner_validate(t *testing.T) {
 				"checkout.0: Must validate one and only one schema",
 			},
 		},
+		{
```

Comment:
It looks like the validation tests are missing any checks that name and email are required when authorship info is included.